pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Kafka bindings
+description: Learn how to configure Kafka bindings in your AsyncAPI document.
+weight: 300
+---
+
+## Introduction
+
+In a previous tutorial, you learned how to manage schemas with a schema registry. This tutorial teaches you how Kafka bindings function by defining Kafka messages and expanding your AsyncAPI document with protocol-specific details.
+
+## Background context
+
+Bindings are essential for event-driven applications because they provide protocol-specific details, abstracting the complexities of message handling from your application's core logic. They enhance the API's clarity and usability by offering setup options and context for different protocols. Bindings include details like the topics your application reads from or writes to, message formatting, and rules for interacting with multiple data or messages.
+
+In an AsyncAPI document, bindings can be added to various sections like servers, channels, or messages. They contain protocol-specific details unique to each protocol. Binding definitions let you specify functionalities specific to the protocol, which are not covered by AsyncAPI's core features.
+
+You can configure several objects using Kafka bindings. However, for the scope of this tutorial, you will focus on four levels of bindings: [server bindings](https://github.com/asyncapi/bindings/tree/master/kafka#server-binding-object), [operations binding](https://github.com/asyncapi/bindings/tree/master/kafka#operation-binding-object), [channel bindings](https://github.com/asyncapi/bindings/tree/master/kafka#channel-binding-object) and [message bindings](https://github.com/asyncapi/bindings/tree/master/kafka#message-binding-object).
+
+Using the code snippets from the previous tutorial, where you learned [how to manage Avro schemas using a centralized schema registry that enables you to share schemas across multiple applications](pages/docs/tutorials/kafka/managing-schemas-using-schema-registry), you will add configurations for server, operations, channel, and message bindings.
+
```

Comment:
you folks need to be consistent across these tutorials, so they are easy to follow 1-4, but also are useful for someone that just wants to do tut 4

your tutorial is a continuation after tutorial 3, look at it https://github.com/asyncapi/website/pull/2331/files#diff-db9dfd2002015a20cf7de2d3c2ce8e68a1537fde911f556b307e238fc3e1f09bR18. It says that it is next in the line, but also to make it easier for the reader, it shows the final version of the document that was created in tutorial 2. It shows what document will be modified, and then at the end shows modified version

your tutorial shows only final version after you finish tutorial. Would be great to show initial version of the document that you later extend with bindings